packages/api/src/external/carequality/patient.ts

```diff
@@ -46,21 +42,21 @@ export async function discover(
     await processPatientDiscoveryProgress({ patient, status: "processing" });
 
     // Intentionally asynchronous
-    prepareAndTriggerPD(patient, facilityNPI, enabledIHEGW, requestId, baseLogMessage).catch(
+    prepareAndTriggerPD(patient, facilityId, enabledIHEGW, requestId, baseLogMessage).catch(
       processAsyncError(context)
     );
   }
 }
 
 async function prepareAndTriggerPD(
   patient: Patient,
-  facilityNPI: string,
+  facilityId: string,
   enabledIHEGW: IHEGateway,
   requestId: string,
   baseLogMessage: string
 ): Promise<void> {
   try {
-    const pdRequest = await prepareForPatientDiscovery(patient, facilityNPI, requestId);
+    const pdRequest = await prepareForPatientDiscovery(patient, requestId);
```

Comment:
Shouldn't we send this facilityId along so it can be used on `getCqInitiator()`?